modules/openapi-generator/src/main/resources/C-libcurl/api-body.mustache

```diff
@@ -373,7 +373,9 @@ end:
     {{/returnTypeIsPrimitive}}
     if (apiClient->dataReceived) {
         free(apiClient->dataReceived);
+        apiClient->dataReceived = NULL;
     }
+    apiClient->dataReceivedLen = 0;
```

Comment:
Is there any specific reason to keep this outside of the if check? or can it be moved inside also? as it will be already 0 if the if statement is false. So the line becomes obsolete.

Reply:
Yes. it can be moved inside. I will do it.